schema_based_on_diagram.sql.sql

```diff
@@ -1 +1,46 @@
-/**All db schema*/
\ No newline at end of file
+CREATE TABLE patients (
+    id INT PRIMARY KEY,
+    name VARCHAR(250) NOT NULL,
+    date_of_birth DATE
+);
+
+CREATE TABLE history_treatment_id (
+  id INT PRIMARY KEY
+);
+
+CREATE TABLE medical_histories (
+    id INT PRIMARY KEY,
+    FOREIGN KEY (id) REFERENCES history_treatment_id(id),
+    admitted_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+    patient_id INT,
+    FOREIGN KEY (patient_id) REFERENCES patients(id)
+);
```

Comment:
- [ ] Nice job setting up this table, it would be great to add the `status` column too.